### PR TITLE
search on Animixplay

### PR DESCRIPTION
**Pull request by antonydp**

### CODE REVIEW

File: Animixplay/src/main/kotlin/com/hexated/Animixplay.kt [hunk 1]
```diff
@@ -130,9 +131,29 @@ class Animixplay : MainAPI() {
...
+        val searchData = app.post(
+            url = "https://v1.ij7p9towl8uj4qafsopjtrjk.workers.dev",
+            referer = mainUrl,
+            data = mapOf(
+                "q2" to query,
+                "origin" to "1",
+                "root" to "animixplay.to",
+                "d" to "gogoanime.tel")
+        )
+        return Jsoup.parse(JSONObject(searchData.text).get("result").toString()).select("div").map{elem->
```
Comment (hexated) [inline Animixplay/src/main/kotlin/com/hexated/Animixplay.kt:145]: Its better to use parsedSafe in here that provided by cs3, like quicksearch 🙏

File: Animixplay/src/main/kotlin/com/hexated/Animixplay.kt [hunk 2]
```diff
@@ -130,9 +131,29 @@ class Animixplay : MainAPI() {
         }
     }
 
-    override suspend fun quickSearch(query: String) = search(query)
+    
+    override suspend fun search(query: String):List<SearchResponse>?{
+        val searchData = app.post(
+            url = "https://v1.ij7p9towl8uj4qafsopjtrjk.workers.dev",
```
Comment (hexated) [inline Animixplay/src/main/kotlin/com/hexated/Animixplay.kt:137]: Is this constant ?

**Comment by hexated** [comment]
Don't forget to update [extension version](https://github.com/hexated/cloudstream-extensions-hexated/blob/master/Animixplay/build.gradle.kts#L2-L2) increment by 1

**Comment by KillerDogeEmpire** [comment]
@hexated I think it has to stay as it is. I'm not sure though, I will ask antony. It works how it is now, I can make my own PR with the changes you want though.
Screenshot_20221002-192708_AquaStream Beta

**Pull request closed without merging**